### PR TITLE
Workshop Docker image

### PR DESCRIPTION
**Pull request by Yambottle**
This is different from the present `Dockerfile`, so named it as `Dockerfile.workshop`

### CODE REVIEW

**Comment by Yambottle** [comment]
@ttngu207

File: Dockerfile.workshop [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Set environment variable for non-interactive installation
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y
+
+## Install pip
+RUN apt install python3-pip -y
+## Install gitpuller for the potential need when exec lifecycle.hooks postStart
+RUN pip install nbgitpuller
+## Install datajoin dependencies
```
Comment (kabilar) [inline Dockerfile.workshop:15]:
```suggestion
## Install datajoint dependencies
```

File: Dockerfile.workshop [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+RUN apt install graphviz -y
+RUN pip install datajoint matplotlib pydotplus 
+## Put executables in /usr/local/bin/before-notebook.d
+# RUN mkdir /usr/local/bin/before-notebook.d
+# COPY ./file_injector.sh /usr/local/bin/before-notebook.d
+# RUN chmod 775 /usr/local/bin/before-notebook.d/file_injector.sh
```
Comment (kabilar) [inline Dockerfile.workshop:21]: Generally speaking, please remove commented code.

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @Yambottle.  A few thoughts listed below.

- Should we move the file changes in `.datajoint_config.json` and `Dockerfile.workshop` to somewhere in the `vathes` GitHub organization since this is specific to our JupyterHub configuration?  The files within `workflow-array-ephys` are meant for more general community use without our hosted resources.

- Please remove `tests/test_export.py` as the NWB export function in `element-array-ephys` is not quite ready for release.

**Comment by Yambottle** [comment]
> Thanks @Yambottle. A few thoughts listed below.
> 
> * Should we move the file changes in `.datajoint_config.json` and `Dockerfile.workshop` to somewhere in the `vathes` GitHub organization since this is specific to our JupyterHub configuration?  The files within `workflow-array-ephys` are meant for more general community use without our hosted resources.
> * Please remove `tests/test_export.py` as the NWB export function in `element-array-ephys` is not quite ready for release.

This was a long-time-ago PR that I wasn't sure where to put this workshop dockerfile, I'll move it to vathes/dj-ops-tf since that repo is more devops general

**Comment by Yambottle** [comment]
@kabilar I'd recommend to abandon this PR, since everything is so outdated

**Comment by kabilar** [comment]
Thanks @Yambottle.  That sounds like a good plan.  I will close this pull request as many of the remaining changes are in #18.

**Pull request closed without merging**